model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
```

Comment:
but I think Erik is right, we really need the timestamp per open (due to
push). We hammer that into the DB and that gives use oldest and youngest,
no ?

On Tue, Apr 14, 2015 at 3:04 PM, Sebastien Blanc <notifications@github.com>
wrote:

> In
> model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523#discussion_r28323907>
> :
>
> > @@ -36,6 +36,10 @@
> >      private Date submitDate = new Date();
> >      private long totalReceivers;
> >
> > +    private long appOpenCounter;
> > +    private Date firstOpenDate;
>
> firstOpenDate will never be updated once set , remember it's just a
> single instance regardless on how many people received it. The latestOpen
> will be updated each time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/523/files#r28323907>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf